src/components/modebar/index.js

```diff
@@ -12,42 +12,78 @@
 var Plotly = require('../../plotly');
 var d3 = require('d3');
 
+var Icons = require('../../../build/ploticon');
+var buttonsConfig = require('./buttons_config');
```

Comment:
But isn't that correcting an issue that is caused by splitting things out this much anyway? This wasn't an issue in the original design. It begs the question.